case-lib/relay.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2021-2025 Intel Corporation. All rights reserved.
+
+# test-mic_privacy.sh needs to control mic privacy settings (on/off)
+# needs usbrelay package: https://github.com/darrylb123/usbrelay
+# param1: switch name
+# param2: switch state
+usbrelay_switch()
+{
+    # Declare a constant for the relay settle time
+    USBRELAY_SETTLE_TIME=0.5
+
+    local switch_name=$1
+    local state=$2
+
+    dlogi "Setting usbrelay switch $switch_name to $state."
+    usbrelay "$switch_name=$state" > /dev/null || {
```

Comment:
```suggestion
    usbrelay "$switch_name=$state" || {
```
as already discussed https://github.com/thesofproject/sof-test/pull/1278#discussion_r2202016464